nifi-registry/nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/provider/flow/git/GitFlowPersistenceProvider.java

```diff
@@ -94,9 +94,12 @@ public void onConfigured(ProviderConfigurationContext configurationContext) thro
             flowStorageDir = new File(flowStorageDirValue);
             final boolean localRepoExists = flowMetaData.localRepoExists(flowStorageDir);
             if (remoteRepo != null && !remoteRepo.isEmpty() && !localRepoExists){
+                logger.info("Validating remote repo '%s'...", remoteRepo);
```

Comment:
Placeholder values for logging should be `{}` instead of `%s`, also recommend the wording adjustment:
```suggestion
                logger.info("Validating remote repository [{}]", remoteRepo);
```